cloud/pkg/apis/edgeclusters/v1/types.go

```diff
@@ -98,11 +98,11 @@ type EdgeClusterSpec struct {
 	Labels map[string]string `json:"labels,omitempty"`
 }
 
-// EdgeClusterStatus is a description of Mission status
-type EdgeClusterStatus struct {
+// EdgeClusterState is a description of Mission status
+type EdgeClusterState struct {
 	HealthStatus string `json:"healthstatus,omitempty"`
 
-	EdgeClusters []string `json:"edgeclusters,omitempty"`
+	EdgeClusters map[string]string `json:"edgeclusters,omitempty"`
```

Comment:
Clusters stand for array. If it is map, it might be EdgeClusterMap

Reply:
I renamed it to SubEdgeClusterStates